src/Queue/QueueFactory.php

```diff
@@ -15,11 +16,19 @@ class QueueFactory {
 
   const SERVICE_NAME = 'queue.rabbitmq';
```

Comment:
Easier to understand than a bare string, when reading code, IMO, so won't change.